src/sparse_blas/backends/mkl_common/mkl_spsv.cxx

```diff
@@ -45,16 +43,16 @@ sycl::event release_spsv_descr(sycl::queue& queue, spsv_descr_t spsv_descr,
     return detail::submit_release(queue, spsv_descr, dependencies);
 }
 
-void check_valid_spsv(const std::string& function_name, oneapi::mkl::transpose opA,
-                      matrix_view A_view, matrix_handle_t A_handle, dense_vector_handle_t x_handle,
+void check_valid_spsv(const std::string& function_name, transpose opA, matrix_view A_view,
```

Comment:
much simpler now to distinguish between oneMKL product types and the oneMATH types :)  good changes for this interfaces project